### DISCUSSION TITLE
adguardhome upgrade

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#6034] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

hi 

lookin for update versiion automaticaly

some times when i log on my adguard home instance, i can see a upper banner that notices me an update, then i click, update and the instance reload. ok great.

does a sh script checking adghome update exists ?
if yes, where i can find it and in a schedule task at what frequency can i ask to update ?
 
Thanks 

### Proposed solution

-

**Comment by ainar-g** [comment]
Please do not check

> I want to request a feature or enhancement and not ask a question

and

> I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

checkmarks blindly.  They are tips on how to find answers quickly.

You can upgrade AdGuard Home from the command line using the `--update` command-line flag, as [documented on the Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#command-line).
